crates/tabby-common/src/api/event.rs

```diff
@@ -35,30 +35,32 @@ pub enum Event {
     View {
         completion_id: String,
         choice_index: u32,
-
         #[serde(skip_serializing_if = "Option::is_none")]
         view_id: Option<String>,
+        #[serde(skip_serializing_if = "Option::is_none")]
+        user_agent_info: Option<String>,
```

Comment:
please add field `user_agent` **only** to `Completion` event, and extract `User-Agent` from routes/completion.rs

Reply:
sure I will do it tonight

Reply:
I will separate the two PRs